README.md

```diff
@@ -27,6 +27,34 @@ poetry shell
 
 in which you can run all of the following commands. Alternatively you can prefix all of the following commands with `poetry run`.
 
+### Install PostgreSQL
```

Comment:
I'd suggest mentioning that this is a requirement for running the unit tests due to the dependency on testing.postgresql https://github.com/tk0miya/testing.postgresql